pkgs/by-name/en/ente-web/package.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchYarnDeps
+, nodejs
+, yarnConfigHook
+, yarnBuildHook
+
+}: stdenv.mkDerivation (finalAttrs: rec {
```

Comment:
Could you please use `nixfmt-rfc-style` package?

Reply:
done.